db/migrate/20190312181641_migrate_email_user_options.rb

```diff
@@ -0,0 +1,26 @@
+class MigrateEmailUserOptions < ActiveRecord::Migration[5.2]
+  def up
+    add_column :user_options, :email_level, :integer, default: UserOption.email_level_types[:only_when_away], null: false
+    add_column :user_options, :email_messages_level, :integer, default: UserOption.email_level_types[:always], null: false
+
+    execute <<~SQL
+      UPDATE user_options
+      SET email_level = CASE
+        WHEN email_direct IS TRUE AND email_always IS TRUE
```

Comment:
In SQL you don't need `IS TRUE`, you can do a comparison like `WHEN email_direct AND (NOT email_always)`. Just a little shorter :)